app/components/authentication_tokens/details/view.rb

```diff
@@ -53,6 +56,10 @@ def rows
 
         row
       end
+
+      def revokable?
+        @revokable ||= AuthenticationTokenPolicy.new(current_user, token)
```

Comment:
This doesn't seem right as it will return an instance of `AuthenticationTokenPolicy` and will evaluate to `true`